eZ/Publish/Core/FieldType/Validator/IntegerValueValidator.php

```diff
@@ -45,7 +45,12 @@ public function validateConstraints($constraints)
             switch ($name) {
                 case 'minIntegerValue':
                 case 'maxIntegerValue':
-                    if ($value !== false && !is_integer($value)) {
+                    // @deprecated false constraint value, see https://jira.ez.no/browse/EZP-26382
```

Comment:
@joaoinacio specifically @bdunogier is asking for something like:
https://github.com/ezsystems/ezpublish-kernel/pull/1801/files#diff-e762cd40d2f3fd134400dd83c5032de5R45